packages/shared/src/hooks/feed/useFeedVotePost.ts

```diff
@@ -114,28 +132,30 @@ export default function useFeedUpvotePost(
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [items, updatePost]);
 
-  return async (post, index, row, column, upvoted): Promise<void> => {
-    if (!user) {
-      showLogin(AuthTriggers.Upvote);
-      return;
-    }
-    trackEvent(
-      postAnalyticsEvent(postEventName({ upvoted }), post, {
-        columns,
-        column,
-        row,
-        ...feedAnalyticsExtra(feedName, ranking),
-      }),
-    );
-    if (upvoted) {
-      await upvotePost({ id: post.id, index });
-      if (setShowCommentPopupId) {
-        requestIdleCallback(() => {
-          setShowCommentPopupId(post.id);
-        });
+  return {
+    onUpvote: async (post, index, row, column, upvoted): Promise<void> => {
```

Comment:
Should it not also export onDownvote?

Reply:
Downvote should be available from the feed, right? Through the post card options menu.

Reply:
Yeah, but there I am using other hook to avoid some props drilling. Because for feed hook you need items too.